lib/NightwaveChallenge.js

```diff
@@ -23,7 +23,7 @@ class NightwaveChallenge extends WorldstateObject {
      * Whether or not this is a daily challenge
      * @type {Boolean}
      */
-    this.isDaily = data.Daily;
+    this.isDaily = data.Daily || false;
```

Comment:
could you add a test that asserts it's not undefined and is false?

Reply:
Sure I can try